src/olympia/devhub/tasks.py

```diff
@@ -660,8 +728,10 @@ def resize_preview(src, instance, **kw):
     try:
         (sizes['thumbnail'], sizes['original']) = resize_image(
             src, thumb_dst, amo.ADDON_PREVIEW_SIZES[0])
+        pngcrush_image(thumb_dst)
         (sizes['image'], _) = resize_image(
             src, full_dst, amo.ADDON_PREVIEW_SIZES[1])
+        pngcrush_image(full_dst)
```

Comment:
Maybe call `pngcrush` directly inside `resize_image` at some point? That'd imply doing it for user images and collection icons but maybe that's not a bad idea either. Can be a follow-up though